CommonHelp.cmake

```diff
@@ -0,0 +1,76 @@
+# Copyright (c) 2017 Stefan.Eilemann@epfl.ch
+#                    Raphael.Dumusc@epfl.ch
+#
+# Generate help page for doxygen by running Name application with --help:
```

Comment:
Yes, this was not clear. I added to the Arguments list:
   Name: an existing application target